bot/resources/tags/venv.md

```diff
@@ -0,0 +1,18 @@
+**Virtual Environments**
+
+Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible.
+
+To create a new virtual environment, you can use the `venv` module: `python -m venv .venv`
+
+Then, to activate the new virtual environment:
+
+PowerShell: `.venv\Scripts\Activate.ps1`
+Bash: `source .venv/bin/activate`
+
+Note: On Windows, you may need to change the [execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies) first:
```

Comment:
This sort of feels a little out of scope for the tag. Would be nice if we could just link to this section in the venv docs next to the command above, although i'm not sure you can specifically link to that part. You could try linking to another page, like https://dev.to/aka_anoop/enabling-virtualenv-in-windows-powershell-ka3, may be nicer to have this info on our site at some point though,

Reply:
The issue with those pages is that they suggest changing the execution policy to `Unrestricted`, whereas `RemoteSigned` would be preferred -- as the Python docs suggest.

Until we get a page for the site, keeping something about execution policies here will probably be useful.

Reply:
Another possible option could be to link to a gist, something like https://gist.github.com/wookie184/4bec5f2785ad72736a6a46bb4cc7defc#file-executionpolicy-md, although if you'd like to leave the note in the tag for now I think that's fine too.

Reply:
I don't think length is a problem, but I do think the prominence is a problem. Someone reading the tag might think that this is immediately important/applies all the time, given the font-size of codeblocks and the fact it comes before the very end of the tag, and this isn't strictly true, you only need to run the command once. It takes a lot of mindshare and I think that can be improved. 

Try moving it to the very, very end of the tag, as well as sticking it within \`regular code tags\` rather than a \`\`\`code block\`\`\`, and see if that's better?

Reply:
My reasoning for its placement is that it follows directly after the activation examples, which is the stage where changing the execution policy is required. I'll make some changes to suggest that it's only required once.

I agree that it is rather prominent, however. Changing to single backticks seems to work, but it should probably be on a new line to avoid wrapping.